### PR TITLE
24840 position formatting popup column header

### PR DESCRIPTION
**Pull request by npfitz**
EPIC #24840 

PR to convert `ChartClickAction` popovers to Tippy, as well as adjust positioning of the popover when adjusting the column formatting so you can see what you're doing.

chrome_cdNj9o6CSZ

### CODE REVIEW

**Review by alxnddr — APPROVED** [review]
Codewise looks good! Seems like you need to tweak TippyPopover to remove the delay when you click on a column header. There is a delay between a click and tooltip appearance:

<video src="https://user-images.githubusercontent.com/14301985/188977678-4649e32e-bd17-4703-b6d7-97e1f0cb97a6.mov" />

Approving in advance, great job 👍

**Comment by npfitz** [comment]
Good eye! I'll take a look and see if I can remove the delay

File: frontend/src/metabase/visualizations/components/ChartClickActions.jsx [hunk 1]
```diff
@@ -183,92 +198,105 @@ class ChartClickActions extends Component {
...
-              </div>
-            ))}
-          </div>
-        )}
-      </Popover>
+              ))}
+            </div>
+          )
+        }
+        {...popoverAction?.popoverOptions}
```
Comment (WiNloSt) [inline frontend/src/metabase/visualizations/components/ChartClickActions.jsx:298]: Q: Is this more like `popoverProps`? I'd expect `options` would be something that you pass as a whole object, but spreading everything in `popoverOptions` make it seems like this behave more like a React component props. What do you think?

**Review by WiNloSt — APPROVED** [review]
🎉  Nick this is looking great! I have some thing to note about the way we now position column options on mobile screen as It's now harder to see column options on mobile devices.

By the way, I don't think we should fix anything yet, this is more like a question for our designers.


 
#### Before
Screen Shot 2022-09-08 at 1 51 59 PM
#### After
Screen Shot 2022-09-08 at 1 52 14 PM

File: frontend/src/metabase/visualizations/components/ChartClickActions.jsx (hunk 1 above)
Comment (npfitz) [reply frontend/src/metabase/visualizations/components/ChartClickActions.jsx]: Agreed! Changed to `popoverProps`

**Pull request merged**